Gruntfile.js

```diff
@@ -7,12 +7,20 @@ var CONTENT_PATH_PREFIX = "content/";
 module.exports = function(grunt) {
 
     grunt.registerTask("hugo-build", function() {
+        done = this.async();
```

Comment:
Horrors of javascript's past still haunt it.. `done =` is valid but not good practice. It's not a big deal because of the scope of the script but I would change it to `const done = this.async();`